rootfs/templates/wrapper

```diff
@@ -89,6 +89,11 @@ function use() {
     if [ "${HOME}" == "/localhost" ]; then
       echo "WARNING: not mounting ${HOME} because it conflicts with geodesic"
     else
+      if [[ $(uname -r) =~ Microsoft$ ]]; then
+        WSL_USER_DIR=${WSL_USER_DIR:-$(ls -d /mnt/c/Users/* | head -1)} 
+        WSL_USER_NAME=${WSL_USER_NAME:-user} 
+        HOME="${WSL_USER_DIR}/AppData/Local/Packages/CanonicalGroupLimited.Ubuntu18.04onWindows_79rhkp1fndgsc/LocalState/rootfs/home/${WSL_USER_NAME}"
```

Comment:
> `CanonicalGroupLimited.Ubuntu18.04onWindows_79rhkp1fndgsc` 

This part seems highly unique. I hope we can find an alternative approach.

Reply:
I think it is okay. Just name of the company and app. Added regular expression for version and some id(maybe) at the end.